README.md

```diff
@@ -113,7 +123,7 @@ end
 Your answer:
 ```text
 
-Replace this with your answer
+Both "describe" and "context" do the same thing in rspec; they provide organization to the structure of the tests. They can be nested within each other, but they don't affect the code.  It helps to add some visual context to the method being tested.
 ```
```

Comment:
`describe` tells you what specific unit is being tested. `context` tells you under what conditions a unit is being tested.